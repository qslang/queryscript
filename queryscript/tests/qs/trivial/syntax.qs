fn sum2<R>(a R) -> External<SumAgg<R>> = sql;
