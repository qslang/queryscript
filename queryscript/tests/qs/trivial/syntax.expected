{
    "compile_errors": [],
    "decls": {
        "let sum2": ∀ "R" λ {
        	a "R",
        } -> External<SumAgg<"R">>,
    },
    "queries": [],
}